src/inventory_ui.cpp

```diff
@@ -80,7 +80,16 @@ class selection_column_preset: public inventory_selector_preset
             } else if( available_count != 1 ) {
                 res << available_count << ' ';
             }
-            res << entry.location->display_name( available_count );
+            if (entry.location->ammo_type() == "money") {
+                if (entry.chosen_count > 0 && entry.chosen_count < available_count) {
+                    res << entry.location->display_money(entry.chosen_count, entry.location.charges_in_stack(entry.chosen_count)) 
+                        << string_format(" of ($%.2f)", (double)entry.location.charges_in_stack(available_count) / 100);
```

Comment:
This needs to be translated. Also add the first part of the result (what `display_money` returns) to the translatable string, so translators can change the order: `_( "%s of ($%.2f)" ), display_money( ... ), (double)...`.

Reply:
Please check the current version, I'm not sure I'm handling the to-be translated texts correctly.